dask/array/tests/test_routines.py

```diff
@@ -489,15 +489,13 @@ def test_bincount_with_weights():
     assert same_keys(da.bincount(d, weights=dweights, minlength=6), e)
 
 
-def test_bincount_raises_informative_error_on_missing_minlength_kwarg():
-    x = np.array([2, 1, 5, 2, 1])
```

Comment:
Does this test still apply? If so, would it be possible to retain it? If not, why does it now not work?

Reply:
It no longer applies, the informative error has been replaced with the new behaviour (`_bincount_sum` works out what minlength should be and combines the ragged numpy arrays).

If this question is about why I've changed the input array values, that's so there can be no confusion between the maximum value in the array (which is related to minlength) and the number of elements in the array (which is unrelated).